src/Microsoft.Extensions.DependencyInjection.Ordered/OrderedServiceDescriptor.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.DependencyInjection.Ordered
+{
+    public class OrderedServiceDescriptor: EnumerableServiceDescriptor
```

Comment:
I meant could we call this `OrderedEnumerableServiceDescriptor` in keeping with the naming convention for the IEnumerable?